students/brian_warn/session04/kata_test.py

```diff
@@ -0,0 +1,94 @@
+#!/Library/Frameworks/Python.framework/Versions/3.6/bin/python
+# Kata Fourteen
+import random
+
+
+def main():
+    exceptions = ['.', '?', ' ', '!', ',', '-']
+
+    def build_trigram(infile):
+        ''' Function that builds the trigram dictionary from the source file. '''
+
+        trigrams = {}
+        with open(infile) as sherlock:
+            sherlock.readline()
+            for line in sherlock:
+                line = line.rstrip('\r\n')
```

Comment:
.strip(), .rstrip(), etc will strip any whitespace by default -- so better not to specify \r\n, jsut in case there are spaces or tabs, or ??? in there too.

Reply:
Makes sense now, @PythonCHB.